kms/keys.go

```diff
@@ -0,0 +1,236 @@
+package kms
+
+import (
+	"go.uber.org/zap"
+	"google.golang.org/api/cloudkms/v1"
+)
+
+type CryptoKey struct {
+	ID               uint `gorm:"primarykey"`
+	ProjectID        string
+	CreateTime       string
+	Name             string
+	NextRotationTime string
+
+	PrimaryAlgorithm string
+
+	CaviumCerts          []*CryptoKeyCertificateChainsCaviumCerts          `gorm:"constraint:OnDelete:CASCADE;foreignKey:ID"`
+	GoogleCardCerts      []*CryptoKeyCertificateChainsGoogleCardCerts      `gorm:"constraint:OnDelete:CASCADE;foreignKey:ID"`
```

Comment:
I think those table names are too long (try test it with postgres). I think we will need to cut it short for example - `KeyCertGoogleCardCert`